sky/serve/service_spec.py

```diff
@@ -250,12 +264,16 @@ def autoscaling_policy_str(self):
                 f'{self.target_qps_per_replica})')
 
     def __repr__(self) -> str:
+        lb_policy_str = ''
+        if self.load_balancing_policy is not None:
+            policy_name = self.load_balancing_policy
+            lb_policy_str = f'\nLoad balancing policy:           {policy_name}'
         return textwrap.dedent(f"""\
             Readiness probe method:           {self.probe_str()}
             Readiness initial delay seconds:  {self.initial_delay_seconds}
             Readiness probe timeout seconds:  {self.readiness_timeout_seconds}
             Replica autoscaling policy:       {self.autoscaling_policy_str()}
-            Spot Policy:                      {self.spot_policy_str()}
+            Spot Policy:                      {self.spot_policy_str()}{lb_policy_str}
```

Comment:
```suggestion
            Spot Policy:                      {self.spot_policy_str()}
            Load Balancing Policy:            {self.load_balancing_policy}
```

Seems like this is causing some problems due to the textwrap.dedent. Maybe lets just show the policy name no matter if it is explicit or not

```bash
ssu examples/serve/load_balancing_policies_example.yaml
Service from YAML spec: examples/serve/load_balancing_policies_example.yaml
Service Spec:
            Readiness probe method:           GET /health
            Readiness initial delay seconds:  20
            Readiness probe timeout seconds:  15
            Replica autoscaling policy:       Autoscaling from 2 to 4 replicas (target QPS per replica: 1)
            Spot Policy:                      No spot fallback policy
Load balancing policy:           round_robin
```